contracts/sfc/NodeDriverAuth.sol

```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity 0.8.27;
 
-import {Initializable} from "../common/Initializable.sol";
-import {Ownable} from "../ownership/Ownable.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import {ISFC} from "../interfaces/ISFC.sol";
 import {NodeDriver} from "./NodeDriver.sol";
 import {INodeDriverExecutable} from "../interfaces/INodeDriverExecutable.sol";
 
 /**
  * @custom:security-contact security@fantom.foundation
  */
-contract NodeDriverAuth is Initializable, Ownable {
+contract NodeDriverAuth is Initializable, OwnableUpgradeable {
```

Comment:
Optional: Initializable is included in the OwnableUpgradeable - I am not sure if we want to declare it even through...